rpc/internal/service.go

```diff
@@ -231,15 +231,11 @@ func (srv *perfService) CloseMetrics(ctx context.Context, end *MetricsSeriesEnd)
 }
 
 func addRollups(record *model.PerformanceResult, rollups []*RollupValue) error {
```

Comment:
I think we just need to make a function in the .ext.go file that converts a slice of []*RollupValue to []*model.PerfRollupValue and then we can remove this function entirely.